cylc/flow/prerequisite.py

```diff
@@ -75,6 +79,10 @@ def __init__(self, point):
         # * `False` (prerequisite unsatisfied).
         self._all_satisfied = None
 
+    def __str__(self):
```

Comment:
Why not just use @dataclass and get this for free?

Reply:
I had a quick look, but dataclasses don't work nicely with `__slots__`.